docs/grammar.txt

```diff
@@ -21,12 +21,12 @@ UsingForDeclaration = 'using' Identifier 'for' ('*' | TypeName) ';'
 StructDefinition = 'struct' Identifier '{'
                      ( VariableDeclaration ';' (VariableDeclaration ';')* ) '}'
 
-ModifierDefinition = 'modifier' Identifier ParameterList? Block
+ModifierDefinition = 'modifier' Identifier ParameterList? BlockGeneralAndModifier
```

Comment:
While this change seems to properly restrict the grammar, I don't think it helps in understanding the language.